src/components/DevfilePage/DevfilePageProjects/DevfilePageProjects.tsx

```diff
@@ -65,26 +67,43 @@ export const DevfilePageProjects: React.FC<DevfilePageProjectsProps> = ({
 
   const [errorAlert, setErrorAlert] = useState<null | ErrorAlert>(null);
 
+  const analytics = getAnalytics();
+
   async function triggerDownload(project: Project): Promise<void> {
     setDownloading(true);
-    await download(project).catch((error) => {
+
+    if (analytics) {
+      analytics.track({
```

Comment:
Segment automatically collects users IPs by default which we should avoid by anonymizing IPs on all events using this guidance: https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/identity/#anonymizing-ip.  We should also set the location in the context object by using the browser locale settings since it's another metric we are collecting.